rmf_human_detector/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+cmake_minimum_required(VERSION 3.8)
+project(rmf_human_detector)
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+include(GNUInstallDirs)
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(sensor_msgs REQUIRED)
+find_package(image_geometry REQUIRED)
+find_package(tf2 REQUIRED)
+find_package(tf2_ros REQUIRED)
+find_package(tf2_msgs REQUIRED)
+find_package(tf2_geometry_msgs REQUIRED)
+find_package(cv_bridge REQUIRED)
+find_package(OpenCV REQUIRED)
+find_package(rmf_obstacle_msgs REQUIRED)
+find_package(rmf_building_map_msgs REQUIRED)
+
+if(${OpenCV_VERSION} GREATER 4.0)
+  add_definitions(-D_CV4)
+else()
+  add_definitions(-D_CV3)
+endif()
+
+if(OpenCV_CUDA_VERSION)
+  message (STATUS "OpenCV was compiled with GPU support, use GPU")
+  add_definitions(-D_USE_GPU)
```

Comment:
Again, I don't see this `_USE_GPU` flag used anywhere in the codebase?
The intention for having such definitions added would be to then execute different functions in your code depending on whether cuda was found or not. For example using `cv::cuda::GPUMat` instead of `cv::Mat`